examples/mobile/UIComponents/components/dialogs/dialog.html

```diff
@@ -13,188 +13,154 @@
 	<div class="ui-page" data-role="page" id="demo-popup-page">
 		<div class="ui-header" data-position="fixed">
 			<h1>
-				Popup
+				General Dialogs
 			</h1>
 		</div>
 		<div class="ui-content">
-			<ul class="ui-listview">
-				<li class="ui-group-index">
-					Button
-				</li>
-				<li>
-					<a data-position-to="window" data-rel="popup" href="#1btn_popup">
-						1btn popup
-					</a>
-				</li>
-				<li>
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-anchor">
 					<a data-position-to="window" data-rel="popup" href="#2btn_popup">
-						2btn popup
+						2btn Title Dialog
```

Comment:
Separate numbers before every button. It's easy to recognize this.
Apply to all code.